docs/Makefile

```diff
@@ -5,5 +5,8 @@ IMAGE = grafana/docs-base@sha256:63758b74e3990ab61e274f5e98da092d5c38378829dad04
 docs:
 	docker run -v $(shell pwd)/sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it $(IMAGE) /bin/bash -c 'make server'
 
+docs-no-pull:
+	docker run -v $(PWD)/sources:/hugo/content/docs/grafana/latest -p 3002:3002 --rm -it grafana/docs-base:latest /bin/bash -c 'make server'
```

Comment:
Are you sure you want to use the `latest` revision of the grafana/docs-base image? It recently changed and broke things (which is why the Makefile pins its revision for other targets).

Reply:
this does not work currently, but it's only for local dev work flow maybe @robbymilo can help find out whay we get:

cp themes/grafana/data/webpack_assets.json themes/grafana/static/static
cp: cannot stat 'themes/grafana/data/webpack_assets.json': No such file or directory
Makefile:93: recipe for target 'pre-hugo' failed
make: *** [pre-hugo] Error 1
Makefile:9: recipe for target 'docs-no-pull' failed
make: *** [docs-no-pull] Error 2